### DISCUSSION TITLE
Issue with tailwind vs code extension intellisense

### DISCUSSION THREAD

**Discussion by OmarEssam360** [tailwindlabs/tailwindcss#11583] [Help]
tailwind vs code extension intellisense just stops working when this code is added to `plugins` in `tailwind.config.js`

this code is important because it solves another problem https://github.com/tailwindlabs/tailwindcss/issues/5989#issuecomment-962048436

Any help please ?

![image](https://github.com/tailwindlabs/tailwindcss/assets/87264008/ff64ecf2-492d-4c51-a37d-092fb0c825be)

**Comment by wongjn** [comment]
You could try removing the semi colon (`;`) from the end of the `@apply` in the `.debug-with-outline` rule.

**Reply by OmarEssam360** [reply]
Thank you very much! It worked
